packages/api-security/src/plugins/graphql/userResolvers/PersonalAccessTokens/createPAT.ts

```diff
@@ -0,0 +1,58 @@
+import crypto from "crypto";
+import { NotFoundResponse } from "@webiny/graphql";
+import { ErrorResponse } from "@webiny/commodo-graphql";
+
+const generateToken = (tokenLength = 48) =>
+    crypto
+        .randomBytes(Math.ceil(tokenLength / 2))
+        .toString("hex")
+        .slice(0, tokenLength);
+
+export default async (root, args, context) => {
+    if (!context.user) {
+        return new NotFoundResponse("Current user not found!");
+    }
+
+    const PersonalAccessToken = context.models.SecurityPersonalAccessToken;
+    const User = context.models.SecurityUser;
+
+    const currentUserId = context.user.id;
+    const otherUserId = args.userId;
+
+    const canAssignUser =
+        context.user.access.fullAccess ||
+        context.user.access.scopes.find((scope) => scope === "security:user:crud");
+
+    try {
+        let tokenUserId;
+        if (!otherUserId) {
+            tokenUserId = currentUserId;
+        } else {
+            if (!canAssignUser)
+                return new ErrorResponse({
+                    message:
+                        "Current user is not admin! You must be an admin in order to update other users.",
+                });
+            if (!(await User.findById(otherUserId)))
```

Comment:
One line `if` :)

Sorry for bringing this up, we're definitely gonna need to enforce the ESLint rule here.